cicecore/cicedynB/dynamics/ice_dyn_eap.F90

```diff
@@ -558,7 +558,14 @@ subroutine eap (dt)
                            field_loc_NEcorner, field_type_vector)
       call grid_average_X2Y('U2TF',strocnxT)    ! shift
       call grid_average_X2Y('U2TF',strocnyT)
-
+! shift velocity components from CD grid locations (N, E) to B grid location (U) for transport
```

Comment:
The block of code that follows the call to dyn_finish is growing and appears to be identical for all of the dynamics options (eap, vp, evp).  It would be good (eventually) to put this in its own subroutine (or put it in dyn_finish), to reduce duplication across modules.  Not necessary yet - I will add this to the cleanup issue.